### PR TITLE
Generate README.md File Automatically

### PR DESCRIPTION
**Pull request by khajavi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +0,0 @@
-# ZIO Prelude
```
Comment (sideeffffect) [inline README.md:1]: README.md should still be part of the repository, right?

Comment (sideeffffect) [reply README.md:1]: Oh, so GitHub Action will change code in `master`, correct?

Comment (khajavi) [reply README.md:1]: Yes

**Review by adamgfraser — APPROVED** [review]

**Comment by sideeffffect** [comment]
This is awesome work you're doing @khajavi :superhero: 

It seems like we're not out of the woods yet, though.
https://github.com/zio/zio-prelude/actions/runs/3650675180/jobs/6166952422

Perhaps the protected branch settings need to be tweaked? But those play an important role, right? So I hope there is another way.
Thanks! :pray:

**Pull request merged**